server/controllers/courseController.mjs

```diff
@@ -184,6 +191,20 @@ export const updateCourse = async (req, res) => {
   }
 };
 
+const addFileToParams = async (file, params) => {
+  if (file) {
+    const uploadResult = await cloudinary.uploader.upload(file.path).catch((error) => {
+      console.log(error);
+    });
+
+    params.imageUrl = uploadResult.secure_url;
+    params.logoUrl = uploadResult.secure_url;
+
```

Comment:
Agree, but we need this folder to send this files to the cloud. Since we do not have a lot of files in this folder, I will create clean up scripts in the backend part, if you want, in my next PR.

Reply:
Yes, the folder should stay there, just add the clean-up logic that deletes the binary files after successfully uploading them. If you want, I can also make the logic and push it in this branch.